tests/test_jobs_workers/test_collect_main_files_worker.py

```
@@ -0,0 +1,214 @@
+def test_collect_main_files_processes_multiple_templates(mock_services):
+    """Test processing multiple templates with mixed results."""
+    templates = [
+        TemplateRecord(id=1, title="Template:Test1", main_file=None),
+        TemplateRecord(id=2, title="Template:Test2", main_file="already.svg"),
+        TemplateRecord(id=3, title="Template:Test3", main_file=None),
+    ]
+    mock_services["list_templates"].return_value = templates
+
+    # First template: success
+    # Third template: success
+    def get_wikitext_side_effect(title, project):
```

Ruff (0.14.10)

Unused function argument: `project`

(ARG001)